drivers/timers/arch_alarm.c

```diff
@@ -210,9 +209,7 @@ void weak_function up_timer_getmask(FAR clock_t *mask)
         }
     }
 }
-#endif
 
-#if defined(CONFIG_SCHED_TICKLESS) || defined(CONFIG_CLOCK_TIMEKEEPING)
```

Comment:
should we update clock_systime_timespec to call up_timer_gettime

Reply:
https://github.com/apache/nuttx/pull/15044